test/lib/user.test.js

```diff
@@ -186,4 +191,76 @@ describe('User Model', () => {
                 });
         });
     });
+
+    describe('get tokens', () => {
+        const paginate = {
+            page: 1,
+            count: 50
+        };
+
+        it('has a tokens getter', () => {
+            const listConfig = {
+                params: {
+                    userId: createConfig.id
+                },
+                paginate
+            };
+
+            tokenFactoryMock.list.resolves(null);
+            // when we fetch tokens it resolves to a promise
+            assert.isFunction(user.tokens.then);
+            // and a factory is called to create that promise
+            assert.calledWith(tokenFactoryMock.list, listConfig);
+
+            // When we call user.tokens again it is still a promise
```

Comment:
I'm checking with Petey, this was something I saw in the tests for the other places that we're using getters.

My guess is that we want to make sure the bit where we do `Object.defineProperty` doesn't override the getter, but I'm not sure why we'd need to test for that every time.